scripts/run_ci_checks.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+# Display a help text
+[ "$1" = "-h" -o "$1" = "--help" ] && echo "Runs all CI checks (excluding tests)." && exit
+
+# The path to the YAML file that defines the CI workflows
+YAML_FILE=".github/workflows/lint-and-test.yml"
```

Comment:
We need to make sure this runs when someone is not exactly at the dir root

Reply:
```bash
# Initialize the directory we're searching from (current directory)
current_dir=$(pwd)

# Traverse upwards to find the root directory, assuming it exists somewhere above
while [[ ! -f "$current_dir/sxt-proof-of-sql/.github/workflows/lint-and-test.yml" ]]; do
  # Move up one directory
  current_dir=$(dirname "$current_dir")
  
  # If we reach the root directory (i.e., /), stop to prevent an infinite loop
  if [[ "$current_dir" == "/" ]]; then
    echo "Could not find file."
    exit 1
  fi
done
```

Reply:
Never mind since a script shouldn't be too smart in the sense of trying to locate things unless necessary.